cachi2/core/package_managers/yarn_classic/main.py

```diff
@@ -27,6 +27,26 @@
 
 
 MIRROR_DIR = "deps/yarn-classic"
+_yarn_classic_pattern = re.compile("yarn lockfile v1")  # See [yarn_classic_trait].
```

Comment:
Could we place the link from down below here ?

Reply:
Why do we need regular expressions if the regex pattern is a plain literal? Isn't `"yarn lockfile v1" in ...` more straight forward in this case based on the fact that we read the input file as text in one go?

Reply:
> Could we place the link from down below here ?

I would rather had it down at the end of file unless there is strong opposition to that. I think having a short reference does not pollute the code as much as a long URL: the line will be read more often than the link will be followed, so I chose to make the line easier to read at the expense of making the link slightly harder to follow (now it is just one select-move/chord/click-and-something-unknown away).

> Why do we need regular expressions if the regex pattern is a plain literal?

No good reason. Will amend in the next revision.